plone/app/layout/analytics/view.py

```diff
@@ -7,9 +8,13 @@
 from zope.viewlet.interfaces import IViewlet
 
 
+UNWANTED_TAGS = ["base", "title"]
```

Comment:
The rational is that this is not security but more thinking about what a not so savy site manager, could fuck up :) Off course he/she should handle injecting javascript with care, like it was alway the case with that field in the control panel. 

My code, because it injects in the head section (thanks to @yurj ) I got aware of the fact that we could prevent "user error" simply by filtering only script tag. A bit later I realized we can prevent double base or title tags at a low cost using lxml. 

A pleasant second benefit is that with this code, even if the user makes a typo in the script, the viewlet will render valid HTML.
I'm not familar enough with the control panel code to write up a validator for this head field which would prevent adding base or title tags in the nicest possible way. Anyone would like to assist or give me some pointers?

Hope this helps to understand why it's there.